playbooks/vars/sample-variables-sap-swpm-default-mode-s4hana-distributed-aas-install.yml

```diff
@@ -0,0 +1,56 @@
+---
+
+# Detect SAP NetWeaver software
+sap_install_media_detect_source_directory: "/software"
+sap_install_media_detect_source: local_dir
+sap_install_media_detect_swpm: true
+sap_install_media_detect_hostagent: true
+sap_install_media_detect_igs: true
+sap_install_media_detect_kernel: true
+sap_install_media_detect_webdisp: false
+
+# Product ID for New Installation
+sap_swpm_product_catalog_id: NW_DI:S4HANA2023.CORE.HDB.PD
+#NW_DI:S4HANA2023.CORE.HDB.PD   - Distributed AAS
+#NW_DI:S4HANA2023.CORE.HDB.ABAPHA   - High Availbility AAS
+
+# Generate inifile.params using sap_swpm Ansible Role
+sap_swpm_inifile_list:
+  - swpm_installation_media
+  - credentials
+  - db_connection_nw_hana
+  - nw_config_other
+  - nw_config_additional_application_server_instance
+  - nw_config_host_agent
+  - sap_os_linux_user
+
+# Software
+sap_swpm_software_path: /software/download_basket
+sap_swpm_sapcar_path: /software/download_basket
+sap_swpm_swpm_path: /software/download_basket
+
+# NW Passwords
+sap_swpm_master_password: "NewPass$321"
+sap_swpm_ddic_000_password: "NewPass$321"
+
+# HDB Passwords
+sap_swpm_db_system_password: "NewPass$321"
+sap_swpm_db_systemdb_password: "NewPass$321"
+sap_swpm_db_schema_abap: "SAPHANADB"
+sap_swpm_db_schema_abap_password: "NewPass$321"
+sap_swpm_db_sidadm_password: "NewPass$321"
+
+# NW Instance Parameters
+sap_swpm_sid: S4D
+sap_swpm_aas_instance_nr: "00"
+sap_swpm_aas_instance_hostname: "{{ inventory_hostname_short }}"
```

Comment:
Samples are meant to be static, so please change this to something like "s4h2020-aas"